pkg/services/authn/authnimpl/sync/permission_sync.go

```diff
@@ -0,0 +1,45 @@
+package sync
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/authn"
+	"github.com/grafana/grafana/pkg/util/errutil"
+)
+
+var (
+	errSyncPermissionsFromDBForbidden = errutil.NewBase(errutil.StatusForbidden, "permissions_from_db.sync.forbidden")
+)
+
+func ProvidePermissionsFromDBSync(acService accesscontrol.Service) *PermissionsFromDBSync {
```

Comment:
Nit: Same here "FromDB" does not add anything in my opinion. I think PermissionsSync is fine